tests/testthat/test-qenv_get_var.R

```diff
@@ -24,3 +24,15 @@ testthat::test_that("get_var and `[[` return NULL if object not in qenv environm
   testthat::expect_null(q[["w"]])
   testthat::expect_message(q[["w"]], "object 'w' not found")
 })
+
+testthat::test_that("get_var and `[[` only returns objects from qenv, not parent environment(s)", {
+  q <- qenv()
+
+  testthat::expect_null(get_var(q, "iris"))
+  testthat::expect_null(q[["iris"]])
+
+  qq <- within(q, iris <- head(iris))
+
+  testthat::expect_s3_class(get_var(qq, "iris"), "data.frame")
+  testthat::expect_identical(dim(get_var(qq, "iris")), c(6L, 5L))
+})
```

Comment:
I think these are two separate tests:
1. not accessing variables from parent envs
2. prioritizing this env over parents

However, I find second one irrelevant. As it is obvious that objects in this environments are accessible by `get_var` - expecialy that there is no other environments in the scope.

```suggestion
testthat::test_that("get_var and `[[` does not return objects from parent environments(s)", {
  q <- qenv()

  testthat::expect_null(get_var(q, "iris"))
  testthat::expect_null(q[["iris"]])
})

testthat::test_that("????", {
  q <- within(qenv(), iris <- head(iris))
  testthat::expect_identical(get_var(q, "iris"), head(iris))
})
```

Reply:
Fair enough, removed.